letsencrypt/config.yaml

```diff
@@ -20,7 +20,7 @@ options:
   domains:
     - null
   email: null
-  keytype: ecdsa
+  key_type: ecdsa
```

Comment:
This is an unnecessary breaking change. If people already use that option, we'd need to migrate it. But I feel it is not worth it, I'd suggest to just go with `keytype`.

Reply:
I was using my code from PR #3130 which predated the other naming.